lib/usb/usb_f103.c

```diff
@@ -45,6 +45,8 @@ static void stm32f103_poll(usbd_device *usbd_dev);
 static uint8_t force_nak[8];
 static struct _usbd_device usbd_dev;
 
+#define USBD_PM_TOP 0x40
```

Comment:
"Applications using fewer buffer descriptors could alter PM_TOP" this was the original motive.